### DISCUSSION TITLE
Show resolution messages on compared image file list

### DISCUSSION THREAD

**Discussion by CharlesLiaoO** [WinMerge/winmerge#810] [Ideas]
Can winmerge add a function: Show resolution messages on compared image file list.
Just like the image bellow:
![图片](https://user-images.githubusercontent.com/49112356/120408057-d44d5480-c380-11eb-96dc-31800d18d995.png)

I know there are image resolution message on the bottom status bar when we open image compare tool, but it just not effictient.
![图片](https://user-images.githubusercontent.com/49112356/120408853-6bff7280-c382-11eb-9f71-6a496b96a8b9.png)

Thanks!

**Comment by sdottaka** [comment]
related issue: https://github.com/WinMerge/winmerge/issues/131
